homeassistant/components/axis/device.py

```diff
@@ -142,6 +148,24 @@ async def async_update_device_registry(self):
             sw_version=self.fw_version,
         )
 
+    async def use_mqtt(self, hass: HomeAssistant, component: str) -> None:
+        """Set up to use MQTT."""
+        status = await hass.async_add_executor_job(
+            self.api.vapix.mqtt.get_client_status
+        )
+
+        if status.get("data", {}).get("status", {}).get("state") == "active":
+            self.listeners.append(
+                await mqtt.async_subscribe(hass, f"{self.serial}/#", self.mqtt_message)
+            )
+
+    async def mqtt_message(self, message: Message) -> None:
```

Comment:
This can be a callback since we don't await inside.

Reply:
Will fix!